react-basic-practice/src/components/CustomerTable/index.tsx

```diff
@@ -1,17 +1,58 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import {
   Th,
   Tr,
   Thead,
   Tbody,
   Td,
   Table as ChakraTable,
+  IconButton,
 } from "@chakra-ui/react";
-import { SortingIcon } from "@icons";
-import { ICustomerTableProps } from "@types";
+import { ICustomer, ICustomerTableProps, SortConfigType } from "@types";
 import TableRow from "../TableRow";
+import { SortingIcon } from "@icons";
+import { fetchUsers } from "@services";
+
+const CustomerTable: React.FC<ICustomerTableProps> = ({ columns, action }) => {
+  const [dataCustomer, setDataCustomer] = useState<ICustomer[]>([]);
```

Comment:
customer is enough. But this is an array. It should be plural. I remember I remind you about this case

Reply:
i done